src/lib/playlists.ts

```diff
@@ -94,6 +95,34 @@ export async function loadAllPlaylists() {
   return all_playlists
 }
 
+export async function loadAuthedPlaylistForUser(slug: string) {
```

Comment:
don't like this method name, but the result is an almost instant loading of these props for the user